templates/tests/test-runner.yaml

```diff
@@ -67,10 +67,12 @@ spec:
         - "/bin/sh"
         - "-ec"
         - |
-            export VALUE="{{ .Release.Name }}"
-            consul kv delete _consul_helm_test
-            consul kv put _consul_helm_test $VALUE
-            [ `consul kv get _consul_helm_test` = "$VALUE" ]
-            consul kv delete _consul_helm_test
+            consul members
```

Comment:
Looks good! do we need to run consul members here and then again 2x in the next line?

Reply:
That's a valid point. We could tee the result into a file and performs lines counts against that.

Reply:
Yes, I think that might create more complexity that its worth for just a simple test.. LGTM